src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -34,76 +34,111 @@ const HeaderUserMenu = ({ user }) => {
         </header>
         <div className="user-plan--container">
           <div className="user-plan--type">
-            {user.plan.isSubscribers || user.plan.isMonthlyByEmail ? (
-              <p className="user-plan--monthly-text">
-                <strong>{user.plan.planName}</strong> ({user.plan.maxSubscribers}{' '}
-                {user.plan.itemDescription})
-              </p>
+            {!user.hasClientManager ? (
+              <>
+                {user.plan.isSubscribers || user.plan.isMonthlyByEmail ? (
+                  <p className="user-plan--monthly-text">
+                    <strong>{user.plan.planName}</strong> ({user.plan.maxSubscribers}{' '}
+                    {user.plan.itemDescription})
+                  </p>
+                ) : (
+                  <p className="user-plan--monthly-text">{_('header.plan_prepaid')}</p>
+                )}
+                {user.plan.buttonUrl && user.plan.pendingFreeUpgrade ? (
+                  <button onClick={() => toggleModal(true)} className="user-plan">
```

Comment:
este es el mismo boton que el de abajo ?

Reply:
Junior, that's right, it's the same button, but this code exists since two years. @cbernat  What do you think about merge this two conditionals?: 
![image](https://user-images.githubusercontent.com/21247095/125685428-ddefd8e5-63ce-4001-af37-d1e5e2ad0b0e.png)